package-lock.json

```diff
@@ -0,0 +1,7473 @@
+{
+    "name": "@sayari/sdk",
```

Comment:
These files are auto-generated by the fern generated and should not be modified manually.

Reply:
I shall roll back changes and use the files from main here.